app/src/components/drawer.js

```diff
@@ -1,24 +1,33 @@
-import React from "react";
+import React, { useState } from "react";
 import { NavLink } from "react-router-dom";
 import { ActivitiesIcon, HomeIcon, PeopleIcon, ProjectsIcon } from "../assets/Icons";
 
-const Drawer = () => {
-  return <DrawerDesktop />;
-};
+const Drawer = () => <DrawerDesktop />;
```

Comment:
the changes you've done here are nice to have, but we don't need them in an 'emergency'